### DISCUSSION TITLE
english.all.3class.distsim.crf.ser.gz  ????

### DISCUSSION THREAD

**Discussion by NeilWP** [stanfordnlp/CoreNLP#1412] [Q&A]
The docs and advice I have for  Entity search in a C# class  suggest I install  Stanford.NLP.Core.NLP  and then execute against the 
`
var jarRoot = @"J:\STANDFORD.NLP\stanford-corenlp-4.5.5";
var classifiersDirec = jarRoot + @"\classifiers";
var classifier = CRFClassifier.getClassifierNoExceptions(classifiersDirec + @"\english.all.3class.distsim.crf.ser.gz");
return classifier.classifyToString(text);
`
However in the download neither the .gz file nor the folder  classifiers exist . Help text here is  not very clear
Assistance gratefully received, I cannot be the only person trying to execute an entity look up   in a C# console app.

**Comment by AngledLuffa** [comment]
I don't know anything about C#, nor do I know which C# package you're
using, but I can tell you that our jar files can be downloaded from
https://stanfordnlp.github.io/CoreNLP/

**Comment by NeilWP** [comment]
Hi John,I found the folder/ gz file on Git and its installed in the project. My concern is that I installed NLP using Nuget. As  a package installation , I anticipated that everything I need would be installed. not that I had to hop back to GIT.  My  thanks for your very quick turnaround.  NeilOn 30 Jan 2024, at 02:46, John Bauer ***@***.***> wrote:﻿
I don't know anything about C#, nor do I know which C# package you're

using, but I can tell you that our jar files can be downloaded from

https://stanfordnlp.github.io/CoreNLP/



On Mon, Jan 29, 2024 at 5:38 PM WelshmanOfSteel ***@***.***>

wrote:
